src/test/scala/edu/ie3/simona/test/common/input/PvInputTestData.scala

```diff
@@ -54,4 +54,175 @@ trait PvInputTestData
     Quantities.getQuantity(10, StandardUnits.S_RATED),
     0.95,
   )
+
+  protected val pvSouth1 = new PvInput(
+    UUID.fromString("7ac5bb15-36ee-42b0-902b-9cd520e241b3"),
+    "pv_south_1",
+    new OperatorInput(UUID.randomUUID(), "NO_OPERATOR"),
+    OperationTime.notLimited,
+    nodeInputPvModel3,
+    CosPhiFixed.CONSTANT_CHARACTERISTIC,
+    null,
+    0.2,
+    Quantities.getQuantity(16.09490984119475, StandardUnits.AZIMUTH),
+    Quantities.getQuantity(91.23978812713176, StandardUnits.EFFICIENCY),
+    Quantities
+      .getQuantity(51.75144341774285, StandardUnits.SOLAR_ELEVATION_ANGLE),
+    100,
+    0.9,
```

Comment:
```suggestion
    0.9,
    1.0,
```
kG and kT seem to be unused. But we should transfer them correct.

Reply:
same for the others, I guess (please check this)